finat/alfeld_sorokina.py

```diff
@@ -0,0 +1,42 @@
+import FIAT
+import numpy
+from gem import ListTensor, Literal
+
+from finat.fiat_elements import FiatElement
+from finat.physically_mapped import Citations, PhysicallyMappedElement
+from finat.piola_mapped import piola_inverse
+
+
+class AlfeldSorokina(PhysicallyMappedElement, FiatElement):
+    def __init__(self, cell, degree=2):
+        if degree != 2:
+            raise ValueError("Alfeld-Sorokina only defined for degree = 2")
```

Comment:
Isn't `NotImplementedError` better if it's defined but we haven't got it working yet?